src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/WindowsRuntime/Windows/UI/ViewManagement/InputPane.cs

```diff
@@ -33,19 +32,15 @@ internal class InputPane : IDisposable
             #region Fields
 
             /// <summary>
-            /// The name of the InputPane WinRT runtime class
+            /// Bool to check if the WinRT input pane is supported
             /// </summary>
-            private static readonly string s_TypeName = "Windows.UI.ViewManagement.InputPane, Windows, ContentType=WindowsRuntime";
+            private static readonly bool _isSupported;
```

Comment:
This change seems to turn a capability check into an OS version check.  That's the opposite of what we'd consider best practice.  Is there an overriding reason why we cannot retain the check for the presence of the type itself in some other fashion?

Reply:
Oh I see, is this covered by the EntryPointNotFound below?

Reply:
Yes, this is covered by the `EntryPointNotFoundException` below and by the exception that `RoGetActivationFactory` will throw when it can't find the class.